### PR TITLE
Added SenseCAPT1000A

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/seeed/senseCAPT1000A/uplink.js [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+  };
+}
+
+function isEmpty(obj) {
+  return Object.keys(obj).length === 0;
+}
+
+function consume(event) {
+  const payload = event.data.payloadHex;
+  const result = messageAnalyzed(payload)[0];
```
Comment (mariusgiger) [inline types/seeed/senseCAPT1000A/uplink.js:517]: can there be more than one message in one uplink? if yes iterate

File: types/seeed/senseCAPT1000A/uplink.js [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+  const defaultData = {};
+  const gps = {};
+  const error = {};
+  const lifecycle = {};
+
+  Object.keys(result).forEach((key) => {
+    if (
+      key === "batteryLevel" ||
```
Comment (mariusgiger) [inline types/seeed/senseCAPT1000A/uplink.js:525]: better hardcode this mapping instead of dynamic key mapping

**Review by mariusgiger — COMMENTED** [review]
looks good a few smaller comments

File: types/seeed/senseCAPT1000A/uplink.js (hunk 1 above)
Comment (J2thatsme) [reply types/seeed/senseCAPT1000A/uplink.js:517]: No

**Review by sevyakenza — APPROVED** [review]

**Pull request merged**